proxy/src/main/java/com/velocitypowered/proxy/network/ConnectionManager.java

```diff
@@ -75,6 +78,9 @@ public void bind(final InetSocketAddress address) {
         .childOption(ChannelOption.TCP_NODELAY, true)
         .childOption(ChannelOption.IP_TOS, 0x18)
         .localAddress(address);
+    if (this.epoll) {
+      bootstrap.option(EpollChannelOption.TCP_FASTOPEN, this.server.getConfiguration().getTcpFastOpenMode());
```

Comment:
I totally understand why `TCP_FASTOPEN` should be enabled for client connections (from the proxy to the server), but I'm skeptical on why we should enable it for the server itself, given that Minecraft connections are supposed to be long-lived.

Additionally, I'm not aware of any Minecraft servers which have `TCP_FASTOPEN` on by default, as it's usually disabled. If we merge this in, we should give some instructions on how to make use of fast open on servers.

Reply:
Additionally, this line (and the others like it) will cause a Checkstyle error.

Reply:
>  
> supposed to be long-lived.
> 
Not all the players are playing fair.

Reply:
What do you mean by this?